src/Illuminate/Log/LogManager.php

```diff
@@ -22,7 +22,7 @@
 /**
  * @mixin \Illuminate\Log\Logger
  */
-class LogManager implements LoggerInterface
+class LogManager
```

Comment:
Huge breaking change.

Reply:
@crynobone I thought so too but 1) couldn't find any evidence 2) all tests pass. Just to clarify, LogManager IS NOT a logger, it's a manager class.

There is one hacky way potentially to do this without removing the interface, and somebody did this with the $string parameter already which was supposed to be plain string but Laravel allows an object - we could keep the signature in place and then use func_get_args(), proof of concept:
```php
interface LoggerInterface
{
    public function emergency(string|\Stringable $message, array $context = []): void;
}


class SuperLogger implements LoggerInterface
{
   public function emergency($message, $context = []): void
   {
        print_r(func_get_args());
   }
}

(new SuperLogger)->emergency('Message', 123, 555);
```

just like Laravel already removed "string" from the $message type, if we remove "array" type from the second parameter - it's going to make it a bit more flexible

Reply:
https://github.com/laravel/framework/blob/9.x/src/Illuminate/Foundation/Application.php#L1356

It is being used to resolve `log`, a huge breaking change.

Reply:
Hmm, I went through the code and it's just an alias - it doesn't actually employ PHP interface check, thats why everything works and tests pass.

But it's still kind of wrong I agree, maybe I should just quickly switch to the second option mentioned above